0.2/spec/index.html

```diff
@@ -394,10 +394,26 @@ <h2>Version Directories</h2>
             version directory names.
         </p>
         <p>
-            Version directories MUST have a sub-directory called <code>content</code> if there are files present, and
-            SHOULD NOT contain a <code>content</code> sub-directory otherwise. There MUST be no other files or
-            directories as children of a version directory, other than an <a href="#inventory">inventory file</a>,
-            an <a href="#inventory-digest">inventory digest</a>, and a <code>content</code> directory.
+            Version directories MUST contain a designated content sub-directory if the object contains files to be
+            preserved, and SHOULD NOT contain this sub-directory otherwise. The name of this designated sub-directory
+            MAY be defined in the <a href="#inventory">inventory file</a> using the key "contentDirectory" with the
+            value being the chosen sub-directory name as a string, relative to the version directory. The value of the
+            key "contentDirectory" MUST NOT contain any filesystem delimiter characters. If the key "contentDirectory"
+            is set, it MUST be set in the first version of the object and MUST NOT change between versions of the
+            same object.
+          </p>
+          <p>
+            There MUST be no other files as children of a version directory, other than an <a
+             href="#inventory">inventory file</a> and a <a href="#inventory-digest">inventory digest</a>.
+            The version directory SHOULD NOT contain any directories other than the designated <code>content</code>
```

Comment:
Potentially remove the `code` element to be consistent with the text in the first sentence.

Reply:
+1